src/components/title.tsx

```diff
@@ -0,0 +1,69 @@
+import createContainer from 'constate';
```

Comment:
I understand why the name of the file here is lowercased - because despite this being a `.tsx` file, there is no traditional react component being exported by this file, rather it's just hooks, containers, etc. 

But to the untrained developer eye, this could be seen as precedence to not capitalize component file names. Just something to note as this codebase evolves.

Reply:
Yeah I wasn't really sure what to call the file since there's 3 things it exports. Good point though.
Maybe we have a separate directory for hooks?